setup.py

```diff
@@ -1,25 +1,35 @@
-from setuptools import setup
+from setuptools import setup, find_packages
 
 #####################################
 VERSION = "0.0.0"
-ISRELEASED = False 
+ISRELEASED = False
 if ISRELEASED:
     __version__ = VERSION
 else:
     __version__ = VERSION + '.dev0'
 #####################################
 
 
+requirements = [
+    'numpy',
+    'scipy',
+    'mbuild>=0.10.4',
+    'unyt>=2.4',
+    'boltons',
+    'lxml',
+]
+
+
 setup(
     name='gmso',
     version=__version__,
+    packages=find_packages(),
+    zip_safe=True,
     author='Matthew W Thompson, Justin Gilmer',
     author_email='matt.thompson@vanderbilt.edu, justin.b.gilmer@vanderbilt.edu',
```

Comment:
Should this author list be expanded to include the major contributors (I don't want to leave Umesh and others out) or should this be modified to be credited to the team? I also assume there is no copyright implication here

Reply:
@mattwthompson I concur there is no copyright implication. You may be interested in the way we've handled this in signac, which attributes the developer group as a whole as well as the "original" author. This [pull request comment](https://github.com/glotzerlab/signac-flow/pull/210#issuecomment-568912170) explains what I did, and you can look at the [file changes](https://github.com/glotzerlab/signac-flow/pull/210/files#diff-2eeaed663bd0d25b7e608891384b7298R40-R43) in that PR as well.